hapi-utils/src/main/java/com/hederahashgraph/fee/SmartContractFeeBuilder.java

```diff
@@ -261,21 +261,21 @@ public FeeData getContractCallLocalFeeMatrices(int funcParamSize,
      *
      */
 
-    int errorMessageSize = 0;   
+    long errorMessageSize = 0;
     int contractFuncResultSize = 0;
     if (contractFuncResult != null) {
-      
+
        if (contractFuncResult.getContractCallResult() != null) {
         contractFuncResultSize = contractFuncResult.getContractCallResult().size();
       }
       if (contractFuncResult.getErrorMessage() != null) {
         errorMessageSize = contractFuncResult.getErrorMessage().length();
-      }      
+      }
     }
-    
-    bpr = BASIC_QUERY_RES_HEADER + getStateProofSize(responseType);    
-   
-    sbpr = BASIC_ENTITY_ID_SIZE + errorMessageSize +  LONG_SIZE  +  contractFuncResultSize;
+
+    bpr =  BASIC_QUERY_RES_HEADER + getStateProofSize(responseType);
+
+    sbpr =  BASIC_ENTITY_ID_SIZE + errorMessageSize +  LONG_SIZE  +  contractFuncResultSize;
```

Comment:
Both lines have an extra space after the assignment operation.

Reply:
Fixed alignment in the file. Thanks !